src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php

```diff
@@ -35,6 +36,7 @@ trait ConfigurationTrait
     private $logLevel;
     private $proxy;
     private $transport;
+    private $grpcInterceptors;
```

Comment:
Same question with the other PR. How do you plan to support this in the config file?

Reply:
There is no plans to support this in the config file.

We could do it in the future if we decide to support config file in `PHP` format to complement or replace the current `INI` format but this is out of scope of this PR.

Reply:
Ditto. Please see the comment in the other PR